### ISSUE TITLE
huge long time to generate trainval GT

### ISSUE THREAD

**Issue by qihao-plus** [FANG-MING/occupancy-for-nuscenes#9]
Thanks for your work.

But I find that it takes huge long time to generate trainval GT even with parallel generating. Also, it takes almost all CPU threads. 

Is that same with your past practice? 

Thanks!

**Comment by FANG-MING** [comment]
Yes, non-keyframe point clouds lack labels, requiring the use of nearest neighbor methods to assign labels to unlabeled points, which can be a time-consuming task.

**Comment by qihao-plus** [comment]
Thanks for your reply. Yes, it takes 96 CPU servers days.   BTW, what’s the difference with CVPR 2023 occupancy GT ? Is that the same method to generate GT as yours?  

Also, the recent OpenOccupancy repo releases it’s code as well for nuscenes.  What’s the difference with your GT? 

Thanks!

**Comment by FANG-MING** [comment]
Yes, this speed is normal. The specific content of OpenOccupancy can refer to their papers.

**Comment by SISTMrL** [comment]
hello, could you please release the generated occupancy data?
